playbooks/switches_config.yml

```diff
@@ -1,14 +1,16 @@
+---
 - name: Switches configuration requirements
-  hosts: 'localhost'
+  hosts: "{{ cifmw_target_host | default('localhost') }}"
   gather_facts: false
   tasks:
```

Comment:
Adding a pre_tasks here, with a `when` condition allowing an early end_play, would be really good.